### PR TITLE
Enable copy/paste for non mac firefox users

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?169570#949638

@millerdev @esoergel

### CODE REVIEW

**Comment by esoergel** [comment]
:+1:

File: src/copy-paste.js [hunk 1]
```diff
@@ -329,27 +330,39 @@ define([
...
             var opts = this.opts().copyPaste;
             vellum = this;
+            isMac = /Mac/.test(navigator.platform);
+
+            function copyModifierKeyPressed(event) {
+                if (isMac && event.metaKey) {
+                    return true;
+                }
+
+                return event.ctrlKey;
```
Comment (millerdev) [inline src/copy-paste.js]: good place for a one-liner? `return (isMac && event.metaKey) || event.ctrlKey`

File: src/copy-paste.js [hunk 2]
```diff
@@ -329,27 +330,39 @@ define([
         init: function () {
             var opts = this.opts().copyPaste;
             vellum = this;
+            isMac = /Mac/.test(navigator.platform);
```
Comment (millerdev) [inline src/copy-paste.js]: This could be moved up to where `isMac` is declared on line 23.

**Comment by millerdev** [comment]
Looks good. Feel free to self-merge unless you want to address my non-blocker comments first.

File: src/copy-paste.js (hunk 1 above)
Comment (emord) [inline src/copy-paste.js]: Oops. I think that was a bug anyways. that would have returned true if it a was a mac and ctrl(command?) key was pressed.

Comment (millerdev) [inline src/copy-paste.js]: oh, good catch

Comment (millerdev) [inline src/copy-paste.js]: although what does that do to Chrome on Mac?

Comment (millerdev) [inline src/copy-paste.js]: tested. works on both FF and Chrome on Mac

**Comment by millerdev** [comment]
tests passed locally

**Pull request merged**